app/keepers.go

```diff
@@ -163,6 +165,46 @@ func InitStakingKeeper(appCodec codec.Codec, keys map[string]*sdk.KVStoreKey, ke
 	return &stakingK
 }
 
+// expandPath resolves paths containing the "~" character to the user's home directory.
+func expandPath(path string) string {
+	if !strings.HasPrefix(path, "~/") {
```

Comment:
cosmos sdk passes `homePath` string as-is without path normalization. using `~` is fairly common to point to the home dir, so explicitly handle that